akka-stream/src/main/scala/akka/stream/impl/fusing/GraphStages.scala

```diff
@@ -44,18 +45,18 @@ final case class GraphStageModule[+S <: Shape @uncheckedVariance, +M](
 /**
  * INTERNAL API
  */
-object GraphStages {
+@InternalApi private[akka] object GraphStages {
```

Comment:
Is the private/internal api below really needed when the surrounding scope is already internal api/private[akka]?